### ISSUE TITLE
upse_time_to_ms mishandles decimals

### ISSUE THREAD

**Issue by Deewiant** [kode54/libupse#1]
Simple test program:

``` c
#include <stdio.h>
#include "upse.h"
int main(int argc, char **argv) {
   for (int i = 1; i < argc; ++i) {
      printf("%ld\n", upse_time_to_ms(argv[i]));
   }
}
```

Buggy results:

```
$  ./test 0:0.1 0:0.10 0:0.100 0:0.1000
100
1000
10000
100000
```

All those time values should be the same: 100, i.e. 0.1 seconds. But e.g. `0:0.1000` is interpreted as 1000 seconds.

**Comment by quackbla** [comment]
The issue is now solved; you may want to mark this as closed if you don't find any other problems with the current version :)

**Comment by kode54** [comment]
Closed by #2
